MADRaS/envs/gym_madras.py

```diff
@@ -123,7 +154,30 @@ def reset(self, prev_step_info=None):
 
         else:
             try:
```

Comment:
@buridiaditya add a line here making a random choice of traffic agents if randomize_traffic==True:
```
if randomize_traffic==True:
    self.traffic_type = [np.random.randint(0, 4) for _ in self.traffic_type]
```